core/templates/dev/head/pages/header.js

```diff
@@ -0,0 +1,6 @@
+$(window).on("load", function() {
```

Comment:
Hi @Tamakimouto, do you think it would be possible to do this using Angular instead, for consistency with the rest of the codebase? You could define a reusable directive, for example.